service/osf_oauth2_adapter/apps.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from django.apps import AppConfig
+
+
+class OsfOauth2AdapterConfig(AppConfig):
+    name = 'osf_oauth2_adapter'
+    # staging by default so people don't have to run OSF to use this.
+    osf_api_url = os.environ.get('OSF_API_URL', 'https://staging-api.osf.io').rstrip('/') + '/'
```

Comment:
Yeah, it looks like if you forget the trailing slash in `'OSF_API_URL'` this adds it back